### PR TITLE
Serve versioned cards from the website

### PR DESCRIPTION
**Pull request by Zhouzi**
This PR updates scripts to create a dump to be served from the website.

* [x] Create a short dump of the cards in /dist/website
* [x] Create an MD5 of the cards to serve as the version
* [ ] Update the core logic to be fed with a dump instead of importing a static JSON file

This PR does not intend to move the JSON file of the cards out of the extension but is a first step. I'll open another PR to make the relevant requests, store the version/cards & all from the extension.

The first commit of this PR replaces the current Eslint configuration with GitBook's. I was getting mad with formatting 😛  You can skip the first commit to see the relevant changes.

### CODE REVIEW

File: scripts/buildDictionary.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    })
+    .then(() => {
+        spinner.text = 'Fetching cards from the Gwent API...';
+        return GwentAPI.cards({ offset: 0, limit: 9999 });
+    })
+    .then(({ results }) => results)
+    .then(cards =>
+        cards.reduce(
+            (promise, card, index) =>
+                promise.then(acc =>
```
Comment (Soreine) [inline scripts/buildDictionary.js]: We could use pLimit here to work at a higher concurrency rate
https://www.npmjs.com/package/p-limit

File: scripts/buildDictionary.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                    }).then(fullCard => {
+                        spinner.text = `Downloading the cards: ${index}/${cards.length}`;
+                        return acc.concat([fullCard]);
+                    })
+                ),
+            Promise.resolve([])
+        )
+    )
+    .then(cards => {
+        spinner.text = 'Reducing cards disk size...';
```
Comment (Soreine) [inline scripts/buildDictionary.js]: Ideally, we should also serve the gzipped version of cards ^^ but it's not possible using gh-pages.

**Comment by Zhouzi** [comment]
Let's rethink our approach with homecoming in mind. I'm closing this PR 👌

**Comment by Soreine** [comment]
I have merged it though in the end.

**Pull request closed without merging**